packages/git/src/browser/style/index.css

```diff
@@ -343,9 +405,9 @@
     cursor: pointer;
 }
 
-.git-theia-header:hover > .git-change-list-buttons-container {
+.git-theia-header:hover>.git-change-list-buttons-container {
```

Comment:
Minor, but I'm not sure the removal of the spacing here is required.
Throughout Theia, we use this spacing in other css files.

Reply:
@vince-fugnitto Thanks for looking at this.  Yes, something looks very wrong there.  I was aware it looked odd as the section got too small but I have not seen that.  I will do further testing on Linux (I developed it on Windows).

Reply:
The removal of the space was unintentional  and I did not notice it.  I'll restore it.